yang/vyatta-protocols-frr-zebra-v1.yang

```diff
@@ -0,0 +1,227 @@
+module vyatta-protocols-frr-zebra-v1 {
```

Comment:
You have a mixture of tabs and spaces in there. Please stick to tabs, and keep lines to 80 characters wide when the tabs are represented as 4 spaces wide.

Reply:
The file has been converted to tabs